BE/FunctionalEntities/FunctionalEntities.rdf

```diff
@@ -132,21 +111,113 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>functional entity</rdfs:label>
-		<skos:definition>any independent party (i.e., person, business entity, governmental entity, or other organization) defined in terms of its function</skos:definition>
+		<skos:definition>independent party defined in terms of its function</skos:definition>
```

Comment:
but it's not an independent party - it's one of potentially many individuals related to an independent party, one for each of its functions*.  This definition is utterly misleading.
* (except that a single individual may be classified by multiple FunctionalEntity subclasses, according to some undocumented scheme)